examples/kubernetes/plugins/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
Could you also update the command references at http://cilium.readthedocs.io/en/latest/gettingstarted/minikube/#extra-metrics

Reply:
thanks 👍